src/Kros.ProblemDetails.Extensions/ProblemDetailsExtensions.cs

```diff
@@ -0,0 +1,43 @@
+﻿using FluentValidation;
+using Hellang.Middleware.ProblemDetails;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+using System;
+
+namespace Kros.ProblemDetails.Extensions
+{
+    /// <summary>
+    /// Hellang problem details service extensions.
+    /// </summary>
+    public static class ProblemDetailsExtensions
+    {
+        /// <summary>
+        /// Registers Hellang problem details to IoC container.
+        /// </summary>
+        /// <param name="services">IoC container.</param>
+        /// <param name="environment">Current environment.</param>
+        /// <param name="configAction">Additional action for configuring Hellang problem details.</param>
+        public static IServiceCollection AddCustomProblemDetails(
+            this IServiceCollection services,
+            IWebHostEnvironment environment,
+            Action<ProblemDetailsOptions> configAction = null)
+            =>  services.AddProblemDetails(p =>
+            {
+                p.IncludeExceptionDetails = (context, ex) => IncludeExceptionDetails(environment, ex);
```

Comment:
Premenná `environment` nie je potrebná, zjednoduší sa tak používanie, keď nebude musieť človek zadávať hodnotu ručne. Treba použiť `context` a prvú časť podmienky získať z neho.

``` cs
context.RequestServices.GetRequiredService<IHostEnvironment>().IsDevelopment()
```

Reply:
Upravene, s tym, ze som to ponechal aj pre test prostredie, ako som pisal vyssie..